agent-control/src/agent_control/defaults.rs

```diff
@@ -1,17 +1,27 @@
 use crate::agent_type::agent_type_id::AgentTypeID;
+use crate::cli::install::agent_control::AGENT_CONTROL_DEPLOYMENT_RELEASE_NAME;
+use crate::cli::install::flux::AGENT_CONTROL_CD_RELEASE_NAME;
 use crate::opamp::remote_config::signature::SIGNATURE_CUSTOM_CAPABILITY;
 use crate::sub_agent::identity::AgentIdentity;
 use opamp_client::capabilities;
 use opamp_client::opamp::proto::{AgentCapabilities, CustomCapabilities};
 use opamp_client::operation::capabilities::Capabilities;
 
 pub const AGENT_CONTROL_ID: &str = "agent-control";
+
+pub const RESERVED_AGENT_IDS: [&str; 3] = [
+    AGENT_CONTROL_ID,
+    AGENT_CONTROL_CD_RELEASE_NAME,
```

Comment:
is this going to change after Daniel's PR?

Reply:
(having release names configuration forces this not to be a constant)

Reply:
Yes, most probably!